cli/sync_service/README.md

```diff
@@ -0,0 +1,362 @@
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "instanceID": 1578587080905,
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "dataID": 1578587080905,
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "objectSize": 6,
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "chunkSize": 122880
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:   }
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]: ]
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]: Got a new object: lily1
+
+#### List MMS object
+
+hzn mms CLI provides you a command to list your MMS object with given flags. This command will list all objects with its objectID and objectType
+
+    hzn mms object list
+    
+result of the command:
+
+    root@lily-test:~/go/src/github.com/open-horizon/anax# hzn mms object list
+    Listing objects in org userdev:
+    [
+      {
+        "objectID": "policy-basicres.tgz",
+        "objectType": "model"
+      },
+      {
+        "objectID": "policy-multires.tgz",
+        "objectType": "model"
+      },
+      {
+        "objectID": "lily1",
+        "objectType": "model"
+      }
+    ]
+    
+We can specify objectType and objectID to check the object we just published
+
+    hzn mms object list --objectType=model --objectId=lily1
+    
+result of the command:
+
+    root@lily-test:~/go/src/github.com/open-horizon/anax# hzn mms object list --objectType=model --objectId=lily1
+    Listing objects in org userdev:
+    [
+      {
+        "objectID": "lily1",
+        "objectType": "model"
+      }
+    ]
+    
+To show the full information of MMS object metadata, you can add `-l` to the command
+
+    root@lily-test:~/go/src/github.com/open-horizon/anax# hzn mms object list --objectType=model --objectId=lily1 -l
+    Listing objects in org userdev:
+    [
+      {
+        "objectID": "lily1",
+        "objectType": "model",
+        "destinationOrgID": "userdev",
+        "destinationID": "",
+        "destinationType": "",
+        "destinationsList": [],
+        "destinationPolicy": {
+          "properties": [],
+          "constraints": [],
+          "services": [
+            {
+              "orgID": "e2edev@somecomp.com",
+              "arch": "amd64",
+              "serviceName": "my.company.com.services.usehello2",
+              "version": "1.0.0"
+            }
+          ],
+          "timestamp": 1578594025910476311
+        },
+        "expiration": "",
+        "version": "0.0.1",
```

Comment:
Dont show all the internal stuff here. Maybe we need to change the CLI code here too.